test/ingress/ingress.go

```diff
@@ -42,7 +43,7 @@ func GetIngressEndpoint(kubeClientset *kubernetes.Clientset) (string, error) {
 		ingressNamespace = gatewayNsOverride
 	}
 
-	ingress, err := kubeClientset.CoreV1().Services(ingressNamespace).Get(ingressName, metav1.GetOptions{})
+	ingress, err := kubeClientset.CoreV1().Services(ingressNamespace).Get(context.Background(), ingressName, metav1.GetOptions{})
```

Comment:
same re: plumbing ctx thru

Reply:
done in efdb1fc3a7cff6eec4d060e3a5744446051894b5